doc/guides/getting_started/src/e2e_regression_setup.md

```diff
@@ -0,0 +1,40 @@
+# ROM E2E Regressions setup
```

Comment:
It would also be useful to have a link pointing to the ROM E2E test plan. Not many people are familiar with how things are setup in the repo.

Reply:
done